datalad/distribution/create.py

```diff
@@ -199,10 +210,31 @@ def __call__(
                                 git_opts=git_opts,
                                 annex_opts=annex_opts,
                                 annex_init_opts=annex_init_opts)
-
+                # record the annex uuid of this repo for this afterlife
+                # to be able to track siblings and children
+                repocfg = vcs.repo.config_reader()
+                if repocfg.has_option('annex', 'uuid'):
```

Comment:
eventually should be RFed to use vcs.uuid?  https://github.com/datalad/datalad/issues/701